source/client-applications.txt

```diff
@@ -4,102 +4,20 @@ Connect BI Tools
 
 .. default-domain:: mongodb
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
-
-Certified Partner Tools
------------------------
-
-- `Tableau Desktop <https://www.tableau.com/products/desktop>`_
-
-- `Microsoft Power BI <https://powerbi.microsoft.com/en-us/>`_
-
-- `IBM Cognos Analytics <https://www.ibm.com/products/cognos-analytics>`_
-
-- `Qlik Sense <https://www.qlik.com/us/products/qlik-sense>`_
-
-- `Looker <https://looker.com/product/business-intelligence>`_
-
-.. note::
-
-   MongoDB supports using the |bi-short| with tools not listed above on 
-   a best-effort basis.
-
-   If you require a custom connector, request one using the 
-   `MongoDB Feedback Engine
-   <https://feedback.mongodb.com/forums/930952-connectors-bi-kafka-spark>`__.
-
-Connection Tutorials
---------------------
-
-You can connect to the |bi-short| using any of these Business
-Intelligence tools.
-
-.. card-group::
-   :columns: 2
-   :style: compact
-   :layout: carousel
-
-   .. card::
-      :cta: Microstrategy
-      :url: https://docs.mongodb.com/bi-connector/current/connect/microstrategy
-      :icon: /images/bi-connector/icons/microstrategy-logo.png
-      :icon-alt: Microstrategy icon
-
-   .. card::
-      :cta: Tableau Desktop
-      :url: https://docs.mongodb.com/bi-connector/current/connect/tableau
-      :icon: /images/bi-connector/icons/tableau-logo.png
-      :icon-alt: Tableau Desktop icon
-
-   .. card::
-      :cta: Microsoft PowerBI
-      :url: https://docs.mongodb.com/bi-connector/current/connect/powerbi
-      :icon: /images/bi-connector/icons/power-bi-logo.png
-      :icon-alt: Microsoft PowerBI icon
-
-   .. card::
-      :cta: QlikSense
-      :url: https://docs.mongodb.com/bi-connector/current/connect/qlik
-      :icon: /images/bi-connector/icons/qliksense-logo.png
-      :icon-alt: QlickSense icon
-
-   .. card::
-      :cta: Microsoft Excel
-      :url: https://docs.mongodb.com/bi-connector/current/connect/excel
-      :icon: /images/bi-connector/icons/excel-logo.png
-      :icon-alt: Microsoft Excel icon
-
-   .. card::
-      :cta: Spotfire Cloud
-      :url: https://docs.mongodb.com/bi-connector/current/connect/spotfire-cloud
-      :icon: /images/bi-connector/icons/spotfire-logo.png
-      :icon-alt: Spotfire Cloud icon
-
-   .. card::
-      :cta: MySQL Client
-      :url: https://docs.mongodb.com/bi-connector/current/connect/mysql
-      :icon: /images/bi-connector/icons/mysql-logo.png
-      :icon-alt: MySQL Client
-
-   .. card::
-      :cta: Looker
-      :url: https://docs.mongodb.com/bi-connector/current/connect/looker
-      :icon: /images/bi-connector/icons/looker-logo.png
-      :icon-alt: Looker icon
+After you install and configure the |bi-short|, you can use |odbc| and
+|jdbc| drivers to connect from many BI Tools and clients. The sections
+below provide basic instructions for connecting to |bi-short| using each
+tool.
```

Comment:
based on the doc it seems as if @sposetti wanted to remove all other tools from this page. Is this correct? If so I can delete the source files for these too.

Reply:
Correct. Wanted to shrink the list of tools that we doc.